### PR TITLE
fix(metrics): getInterstitialToMediaOKJMT calculation

### PR DESCRIPTION
**Pull request by maxinteger**
# COMPLETES [SPARK-465907](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-465907)

## This pull request addresses

`getInterstitialToMediaOKJMT` returned with `undefined` 99.9% of the cases because the media flow did not started because at the time when media engine ready event emitted.

## by making the following changes

used `client.ice.end` latency instead of the media started values. This is what UCF use as well.

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by torgeadelin — APPROVED** [review]
looks good. we should update the wiki

**Review by torgeadelin — CHANGES_REQUESTED** [review]
Actually I've asked in the Join Success space wether this is correct behaviour from Native client. we should wait for the reply before merging this.

**Review by torgeadelin — APPROVED** [review]
got confirmation from CA team and wiki was also updated

**Review by Coread — APPROVED** [review]

**Pull request merged**
